teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, age, name, parent_permission)
```

Comment:
- [ ] Same here. This line is giving an error when I'm trying to create a `Teacher` object with no name because it is supposed to take the default name but here, you have specified the default `name` or `parent_permission` value, Please fix this.

```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
```